src/BenchmarkDotNet.Core/Extensions/StringAndTextExtensions.cs

```diff
@@ -88,5 +88,46 @@ internal static List<string> Wrap(string text, int maxLength)
 
             return lines;
         }
+
+        /// <summary>
+        /// Returns an HTML encoded string
+        /// </summary>
+        /// <param name="s">string to encode</param>
+        internal static string HtmlEncode(this string s)
+        {
+            if (s == null)
+            {
+                return null;
+            }
+
+            StringBuilder sb = new StringBuilder(s.Length);
+
+            int len = s.Length;

```

Comment:
nit: you don't need to save the length in local variable. JIT recognizes it as immutable value and does it for you.